internal/watcher/blocks.go

```diff
@@ -52,6 +52,7 @@ func (bw *BlocksWatcher) Start(ctx context.Context, runner service.Runner) error
 					Time:   time.Now(),
 				}
 				bw.blocksChannel <- ev
+				bw.blocksChannel <- ev
```

Comment:
why are we adding the same event to the same channel twice?

Reply:
In order to listen the data in seperate go routines